endoreg_db/services/video_import.py

```
@@ -158,181 +272,557 @@ def _create_or_retrieve_video_instance(self):
+                        )
+
+                    self.processing_context['anonymization_completed'] = True
+                else:
+                    self.logger.warning("Processed video file not found after move: %s", anonym_target_path)
+            except Exception as exc:
+                self.logger.error("Failed to move processed video to anonym_videos: %s", exc)
+        else:
+            self.logger.warning("No processed video available - processed_file will remain empty")
+
         try:
+            from endoreg_db.utils.paths import RAW_FRAME_DIR
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)